superset/viz.py

```diff
@@ -1901,6 +1901,7 @@ def get_data(self, df):
         if overall:
             df['perc'] = (df.v - min_) / (max_ - min_)
             df['rank'] = df.v.rank(pct=True)
+        df[['x', 'y']] = df[['x', 'y']].fillna('Null')
```

Comment:
Can we use `<NULL>` to align with other places in the code?

Reply:
OK ,I will change it